source/3-landing-page-cicd/cdk/lib/cicd-stack.ts

```diff
@@ -45,10 +46,15 @@ export class LandingPagePipelineStack extends Stack{
                             new PolicyStatement({
                                 actions: [
                                     'organizations:ListAccounts',
-                                    "organizations:ListTagsForResource"
+                                    "organizations:ListTagsForResource",
+                                    "cloudformation:DescribeStacks"
                                 ],
                                 resources: ['*'],
                             }),
+                            new PolicyStatement({
+                                actions: ["sts:AssumeRole"],
+                                resources: [`arn:aws:iam::*:role/cdk-${qualifier}-deploy-role-*`]
```

Comment:
We allow the CodeBuild role to assume the CDK roles in the target accounts to perform the checks.